doc/content/guides/connecting-gateways/mikrotikrouterboard/_index.md

```diff
@@ -0,0 +1,116 @@
+
+![Servers](servers.png)
+
+Edit the server parameters.
+
+1. **Name**: A distinct name 
+2. **Address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+3. **Up port**: UDP upstream port of the Gateway Server, typically 1700.
+4. **Down port**: UDP downstream port of the Gateway Server, typically 1700.
+
+Press OK to save the parameters.
+
+![Server Parameters](parameters.png)
+
+From the top menu, select the **Devices Tab**. Click the **Gateway ID** to configure it.
+
+![Devices](portal-lora-disable.png)
+
+In the device settings, click the **Network Servers Dropdown** to view available servers. Choose the server you just created.
+
+Click OK to save the parameters.
+
+![Device parameters](device-parameters.png)
+
+![Device parameters](device-parameters-selected.png)
+
+Press the **E** to enable the gateway device. It will become a **D** and the device status will update.
+
+![Device enable](portal-lora-enable.png)
+
+If your configuration was successful, your gateway will connect to {{% tts %}} after a couple of seconds.
+
+## Configuration via App
+
+The MikroTik Routerboard exposes a WiFi Access Point (AP) with SSID `MikroTik-xxxxxx`, where `xxxxxx` are the last 6 digits of the device's MAC Address (printed on the back panel).
+
+Using the device where the **MikroTik Mobile App** is installed, connect to this AP. By default, there's no password.
+
+Open the **MikroTik Mobile App**. The connection address is prefilled (ex: `192.168.88.1`). Enter the username and password in the login window. By default, the username is **admin** and there is no password. Select **Connect**.
+
+{{< figure src="mikrotik-routerboard-login.png" alt="MikroTik Routerboard wAP LoRa8 kit Login window" >}}
+
+Once logged in, select the **Gear Box Icon** on the top-right corner of the home page.
+
+{{< figure src="mikrotik-routerboard-home-page.png" alt="MikroTik Routerboard wAP LoRa8 kit Home page window" >}}
+
+Scroll down and click on **LoRa**. In the **LoRa** config window select **Devices**. Select the **LoRa Device** and disable it.
+
+{{< figure src="mikrotik-routerboard-lora-settings.jpeg" alt="MikroTik Routerboard wAP LoRa8 kit LoRa Settings" >}}
+
+{{< figure src="mikrotik-routerboard-lora-disable.jpeg" alt="MikroTik Routerboard wAP LoRa8 kit LoRa disable" >}}
+
+Back in the **LoRa** section, select the **Servers** section. Select the **+** button to add a new server. 
+
+Edit the server parameters.
+
+1. **Name**: A distinct name 
+2. **Address**: Address of the Gateway Server. If you followed the [Getting Started guide]({{< ref "/guides/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+3. **Up port**: UDP upstream port of the Gateway Server, typically 1700.
+4. **Down port**: UDP downstream port of the Gateway Server, typically 1700.
+
```

Comment:
In my case I also had to change the server type to "private" to make it work- it's "public" by default

Reply:
Hmm, can you guys check what that means?

On MultiTech gateways, it has to be the other way around. Private has a different sync word than public, and the LoRaWAN ecosystem uses the public one. Again, in MultiTech terms. So. I'd be curious to what it means on MikroTik and if it's opposite.

Reply:
I didn't have to change it - left public and connected to the enterprise stack. I never routed any packets through it though.

I can't find any documentation from Mikrotik about what it means. I'll send an email.

Reply:
Edit: nevermind, mine was set to private by default.

Reply:
@johanstokking from the [Mikrotik docs](https://help.mikrotik.com/docs/display/ROS/General+Properties)

network (private \| public; Default: public) | Whether sync word should (network=private) or should not (network=public) be used.
-- | --

Reply:
I tried both public and private, both worked

Reply:
@laurensslats did you route packets?